### PR TITLE
Dev hayato patch 1

### PR DESCRIPTION
**Pull request by Dev-Hayato**
Is this how I request a pull request/review? — test...

### CODE REVIEW

**Comment by LemonPocky** [comment]
Please ensure that you use a descriptive title for your branches and pull requests (for example, "homepage-mockup").

Never use parentheses in your file names. Instead, use dashes (-), place them in a separate folder, or add comments.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,274 @@
+<!DOCTYPE html>
+<html lang="en"><head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (LemonPocky) [inline index.html]: Change this to "Studio Ghibli Companion App"

File: index.html [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+      </div>
+    </div>
+  
+  </nav>
+
+    </div>
+    
+  </section>
+
+  <div class="container" style="
```
Comment (LemonPocky) [inline index.html]: What is the purpose of this container? Also, make sure you put styles in the css file.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,274 @@
...
+    display: flex;
+    flex-wrap: nowrap;
+    flex-direction: row;
+    max-width: 1200px;
+    width: 100%;
+    flex-wrap: wrap;
+    justify-content: center;">
+    
+  <div class="container2">
```
Comment (LemonPocky) [inline index.html]: What is the purpose of this container? Give it a name to describe its function.

**Review by LemonPocky — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (Dev-Hayato) [reply index.html]: Complete ✅

File: index.html (hunk 2 above)
Comment (Dev-Hayato) [reply index.html]: Complete ✅ Thank you.

File: index.html (hunk 3 above)
Comment (Dev-Hayato) [reply index.html]: Removed from the html.index file and updated in the .css file.

**Review by Dev-Hayato — COMMENTED** [review]
Requested changes have been made.

File: index.html [hunk 4]
```diff
@@ -50,15 +50,6 @@
...
-  <div class="container" style="
-    display: flex;
-    flex-wrap: nowrap;
-    flex-direction: row;
-    max-width: 1200px;
-    width: 100%;
-    flex-wrap: wrap;
-    justify-content: center;">
     
   <div class="container2">
```
Comment (LemonPocky) [inline index.html]: Give this container a descriptive name, like "movies" or "grid-container".

File: assets/css/home.css [hunk 5]
```diff
@@ -1,4 +1,4 @@
-element.style {
+.container {
```
Comment (LemonPocky) [inline assets/css/home.css]: Does this class still exist in your HTML? Double-check that it does what you want it to do.

**Review by LemonPocky — CHANGES_REQUESTED** [review]

File: assets/css/home.css (hunk 5 above)
Comment (Dev-Hayato) [reply assets/css/home.css]: It no longer exists in the html.  I removed it from the css file as well.

File: index.html [hunk 6]
```diff
@@ -5,6 +5,10 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Studio Ghibli Companion App</title>
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.9.3/css/bulma.min.css">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Titillium+Web:ital,wght@0,400;0,700;1,200&display=swap"
+      rel="stylesheet"
+    />
```
Comment (LemonPocky) [inline index.html]: Any reason to add this? It doesn't seem like this font is used anywhere on this page.

**Review by LemonPocky — CHANGES_REQUESTED** [review]

**Comment by LemonPocky** [comment]
There currently seems to be a merge conflict with main involving index.html. You can overwrite the code in main with your own. Follow the steps below to resolve the conflict after you've made your changes.

File: index.html (hunk 6 above)
Comment (Dev-Hayato) [reply index.html]: No real reason, I was thinking of using google fonts in the future, however, I probably won't have the time to use them.

I'll remove the link and do a git push.

**Review by LemonPocky — APPROVED** [review]

**Pull request merged**